snmp/datadog_checks/snmp/data/profiles/_base_hp.yaml

```diff
@@ -0,0 +1,190 @@
+metrics:
+  # System utilization.
+
+  - # Total time the system has been in full operation.
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.8.1
+      name: cpqHeSysUtilLifeTime
+
+  - # EISA bus utilization as percentage of theoretical maximum during last minute.
+    MIB: CPQHTLH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.8.2
+      name: cpqHeSysUtilEisaBusMin
+
+  # Critical errors.
+
+  - # Overall condition of the critical error log feature.
+    # NOTE: other(1), ok(2), degraded(3), failed(4)
+    MIB: CPQHTLH-MIB
```

Comment:
Seems `CPQHTLH-MIB` shoudl be `CPQHLTH-MIB` (already there before the PR).

Reply:
Wow, good catch — definitely a typo…

Reply:
Good catch- update on this metric and a few others!